cmake/download_onnxruntime.cmake

```diff
@@ -0,0 +1,25 @@
+if(DEFINED TRITON_ONNXRUNTIME_PACKAGE_URL)
+
+  set(DOWNLOAD_PATH "${CMAKE_BINARY_DIR}/_deps/downloads/onnxruntime.zip")
+  set(EXTRACT_DIR "${CMAKE_BINARY_DIR}/_deps/downloads/onnxruntime")
+
+  message(NOTICE "Downloading onnxruntime: ${TRITON_ONNXRUNTIME_PACKAGE_URL}")
+
+  file(DOWNLOAD ${TRITON_ONNXRUNTIME_PACKAGE_URL} ${DOWNLOAD_PATH} SHOW_PROGRESS STATUS DOWNLOAD_STATUS)
+
+  # file(DOWNLOAD ... STATUS DOWNLOAD_STATUS) returns a list with 2 elements
+  list(GET DOWNLOAD_STATUS 0 DOWNLOAD_RESULT)
+
+  if(NOT DOWNLOAD_RESULT EQUAL 0)
+    message(NOTICE "Failed to download: ${TRITON_ONNXRUNTIME_PACKAGE_URL}")
+  else()
+    message(NOTICE "Download successful: ${DOWNLOAD_PATH}" )
+
+    file(ARCHIVE_EXTRACT INPUT ${DOWNLOAD_PATH} DESTINATION ${EXTRACT_DIR} VERBOSE )
+
+    set(TRITON_ONNXRUNTIME_INCLUDE_PATHS ${EXTRACT_DIR}/include)
+    set(TRITON_ONNXRUNTIME_LIB_PATHS ${EXTRACT_DIR}/lib)
```

Comment:
Is this going to break the RHEL build?

Reply:
No unless you start feed the prebuilt asset for RHEL.

Reply:
We may end up using this for our Linux builds in the future, but I suppose we can enhance it when needed.

Reply:
Is it mean that we can resolve this thread?